docker/observation/configs/grafana/provisioning/datasources/datasources.yaml

```
@@ -0,0 +1,58 @@
+    access: proxy
+    uid: tempo
+    url: http://tempo:3200
+    isDefault: false
+    editable: true
+    jsonData:
+      httpMethod: GET
+      tracesToMetrics:
+        datasourceUid: prometheus
+        spanStartTimeShift: -1h
+        spanEndTimeShift: 1h
+        tags: [{ key: 'service.name', value: 'service' }]
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)